pkg/test/framework/components/istio/config.go

```diff
@@ -130,6 +130,10 @@ type Config struct {
 
 	// Expose istiod through ingress, for example for multicluster or VM setups
 	ExposeIstiod bool
+
+	// Indicates whether istiod is using kubeconfig in another cluster
+	// istiodRemote + base running on one cluster, istiod running on the other cluster
+	UsingRemoteKubeConfig bool
```

Comment:
I'm not sure this is the best approach. IIUC the question you're asking is: for a given cluster, where is its source of configuration?  We already do similar things with "topologies" for [network](https://github.com/istio/istio/blob/master/pkg/test/framework/components/environment/kube/flags.go#L216) and [control plane](https://github.com/istio/istio/blob/master/pkg/test/framework/components/environment/kube/flags.go#L210). I think it would make sense to do something similar to make a `configTopology`, where we map each cluster to the cluster that provides its configuration.  By default, every cluster would provide configuration for itself: `0:0,1:1,2:2". However, you could also have clusters share config in a common place, for example: `0:0,1:0,2:0`.

@stevenctl @tbarrella WDYT?

Reply:
Thanks! `configTopology` is a good idea as we may extend this central istiod scenario in the future. This will also make this integration more clear.  I go on this way now.